### PR TITLE
Update documentation on how to use instance IAM roles and set Defaults

### PR DESCRIPTION
**Pull request by ravenbyron**
### Status 

- Documentation update and set defaults

### Description

This updates the documentation to allow the AWS pack to use instance IAM role and also sets the defaults in the config.yaml to use US-EAST-1 and Instance IAM role for connections, this will allow it to work out of the box for anyone in running in US-EAST-1 with a meaningful role assigned. 

### Checklist (tick everything that applies)

- [ ] Metadata: pack.yaml, icon, structure (required for new packs)
- [ *] Version bump (required for changed packs)
- [ ] Code linting (required, can be done after the PR checks)
- [ ] Tests (not required but really recommended)

### CODE REVIEW

**Comment by timoguin** [comment]
I like it.

**Comment by Kami** [comment]
Thanks.

The PR looks good to me.

/cc @DoriftoShoes

File: packs/aws/config.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 setup:
-  region: ""
-  aws_access_key_id: ""
-  aws_secret_access_key: ""
+  region: "us-east-1"
```
Comment (manasdk) [inline packs/aws/config.yaml]: No sure if you intended to check-in `us-east-1`. Typically the config should be empty unless the default value is unambiguous :)

**Comment by manasdk** [comment]
+1

File: packs/aws/config.yaml (hunk 1 above)
Comment (ravenbyron) [inline packs/aws/config.yaml]: Not sure what the standard is here but was trying to go for something that worked for many people out of the box...   I can change if needed

Comment (manasdk) [inline packs/aws/config.yaml]: In my view safer to not have a value specified and have anyone using the pack make an explicit choice.

Comment (ravenbyron) [inline packs/aws/config.yaml]: Changed Back

**Pull request merged**